test/fake/signup_client.go

```diff
@@ -0,0 +1,112 @@
+package fake
+
+import (
+	"encoding/json"
+	"os"
+
+	crtapi "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/testing"
+)
+
+type FakeUserSignupClient struct {
+	Tracker   testing.ObjectTracker
+	Scheme    *runtime.Scheme
+	namespace string
+}
+
+func NewFakeUserSignupClient(namespace string, initObjs ...runtime.Object) *FakeUserSignupClient {
+	clientScheme := runtime.NewScheme()
+	err := crtapi.SchemeBuilder.AddToScheme(clientScheme)
+	if err != nil {
+		log.Error(err, "Error adding to scheme")
+		os.Exit(1)
```

Comment:
Don't exit. Let's panic (at most) if we can't do anything better.

Reply:
Fixed